lib/components/user/user-account-screen.js

```diff
@@ -4,9 +4,9 @@ import { connect } from 'react-redux'
 import { withLoginRequired } from 'use-auth0-hooks'
 
 import { routeTo } from '../../actions/ui'
-import { addOrUpdateUser, isNewUser } from '../../actions/user'
+import { addOrUpdateUser } from '../../actions/user'
```

Comment:
Sorry I didn't comment before, but is a better name for this: `createOrUpdateUser`? I don't feel like we usually use the terminology `add` for this functionality.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/167/commits/5f39e30c920d2355458ff7c4b5e90ad177d104d4